x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/RestListConnectorSyncJobsAction.java

```diff
@@ -40,11 +42,16 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         String connectorId = restRequest.param(ListConnectorSyncJobsAction.Request.CONNECTOR_ID_FIELD.getPreferredName());
         String statusString = restRequest.param(ConnectorSyncJob.STATUS_FIELD.getPreferredName());
         ConnectorSyncStatus status = statusString != null ? ConnectorSyncStatus.fromString(statusString) : null;
+        String jobTypeString = restRequest.param(ConnectorSyncJob.JOB_TYPE_FIELD.getPreferredName());
+        List<ConnectorSyncJobType> jobTypeList = jobTypeString != null
+            ? Arrays.stream(jobTypeString.split(",")).map(ConnectorSyncJobType::fromString).toList()
```

Comment:
Related to my comment above, if we have `fromStringList` in `ConnectorSyncJobType` we could probably reuse it here.